platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/HashSessionIdManager.java

```diff
@@ -69,14 +69,14 @@ public HashSessionIdManager(Random random) {
     /**
      * @return Collection of String session IDs
      */
-    public Collection<String> getSessions() {
+    public synchronized Collection<String> getSessions() {
```

Comment:
I wonder if using ConcurrentHashMap instead would be more performant than synchronizing this method.